packages/ui/components/profile/ProfileVoteButton.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  ChatAltIcon,
+  CheckIcon,
```

Comment:
Is this the same Check used in the designs? or is it close enough?

Reply:
It looks identical to me. The things that are different from the design are the hand and abstain icons. I don't think those icons are from `heroicons` - we may want to see where @modern-edamame is getting those from.